pnpm-lock.yaml

```diff
@@ -6829,7 +6839,6 @@ packages:
     cpu: [x64]
     os: [darwin]
     requiresBuild: true
-    dev: true
```

Comment:
Are you running the same version of pnpm?

Try `corepack enable pnpm` to make sure you are using the same version as this repo